app/build.gradle

```diff
@@ -31,29 +31,31 @@ android {
         }
     }
     assetPacks = [":install_time_asset_pack",":fast_follow_asset_pack_01"]
-    signingConfigs {
```

Comment:
Hey @josikie .  Will you revert?  Per the notes in our chat to allow you to compile